BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -158,15 +158,25 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     publisherSummaryView.tipButton.addTarget(self, action: #selector(tappedSendTip), for: .touchUpInside)
     publisherSummaryView.monthlyTipView.addTarget(self, action: #selector(tappedMonthlyTip), for: .touchUpInside)
     
-    // TODO: Update with actual value below
+    publisherSummaryView.monthlyTipView.isHidden = true
```

Comment:
Is this not hidden by default already? If not carry on…

Reply:
Yeah, it wasn't hidden by default. It was always shown.